Specs/DateTime/Dutch/TimeZoneParser.json

```diff
@@ -224,7 +214,6 @@
     "Context": {
```

Comment:
Is "pacific daylight saving time" how you'd typically reference such timezone in a Dutch sentence?